config.py

```diff
@@ -504,10 +509,12 @@ def configure_android(opts):
     exec_gyp(args + opts['GYP_OPTIONS'])
 
 def configure_win(opts):
+    validate_target_arch(opts)
     validate_windows_tools(opts)
 
     args = core_gyp_args(opts) + ['-Gmsvs_version=' + opts['WIN_MSVS_VERSION']]
-    args += gyp_define_args(opts, {'ms_speech_sdk4': 'MS_SPEECH_SDK4',
+    args += gyp_define_args(opts, {'target_arch':    'TARGET_ARCH',
+                                   'ms_speech_sdk4': 'MS_SPEECH_SDK4',
```

Comment:
I think this line can be removed, as the dependency on SAPI4 is removed earlier in this PR